packages/oid4vci-demo-frontend/src/configs/banqup-account.json

```diff
@@ -1,10 +1,11 @@
 {
   "general": {
-    "credentialName": "Bank account credential",
-    "issueCredentialType": "BankAccount",
-    "baseUrl": "http://192.168.2.90:5000/bank",
+    "credentialName": "Account credential",
```

Comment:
i would remove all verifier stuff from this config as this flow does not have 1

Reply:
Fixed